Tasks/ServiceFabricDeploy/utilities.ps1

```diff
@@ -110,6 +110,41 @@ function Read-PublishProfile
     return $publishProfile
 }
 
+function Get-OverridenApplicationParameters
+{
+    Param (
+        [String]
+        $ApplicationManifestPath
+    )
+
+    $OverrideParameters = @{}
+    $ApplicationManifestXml = [Xml] (Get-Content -LiteralPath $ApplicationManifestPath)
+    foreach ($param in $ApplicationManifestXml.ApplicationManifest.Parameters.Parameter)
+    {
+        $paramName = $param.Name -replace "\.", '_'
+        $paramName = $paramName -replace " ", '_'
+        $paramValue = (Get-Item env:$paramName -ErrorAction Ignore).Value
+        if ($paramValue)
+        {
+            $OverrideParameters.Add($paramName, $paramValue)
+        }
+    }
+
+    return $OverrideParameters
+}
+
+function Get-ApplicationManifesetPath
+{
+    Param (
+        [String]
+        $ApplicationPackagePath
+    )
+
+    $appManifestName = "ApplicationManifest.xml"
```

Comment:
Its always better to assign a variable or constant (with appropriate name) to such file name at the top of the function/file